vispy/visuals/mesh.py

```diff
@@ -171,12 +187,17 @@ class MeshVisual(Visual):
         Shading to use.
     mode : str
         The drawing mode.
+    banded : bool
+        If True apply a banded glsl shader for the colormap
+    nband : int
+        the number of band in the glsl banded shader
     **kwargs : dict
         Keyword arguments to pass to `Visual`.
     """
     def __init__(self, vertices=None, faces=None, vertex_colors=None,
                  face_colors=None, color=(0.5, 0.5, 1, 1), vertex_values=None,
-                 meshdata=None, shading=None, mode='triangles', **kwargs):
+                 meshdata=None, shading=None, mode='triangles', banded=False,
+                 nband=10, **kwargs):
```

Comment:
I'm not a huge fan of all this stuff getting added to Visuals that uses a colormap. I think it would better if users accessed stuff like `nband` and `banded` directly from the colormap object. Something like `my_mesh.cmap.nband = 6`.

Reply:
The above assignment assumes that changing a colormap property automatically triggers an update to the visual using it.

Reply:
Good idea i try to make it and i change example to PyQt4

Reply:
@davidh-ssec  If in MeshVisual tou change the original _clim function in ''float cmap(float val) { return floor((val - $cmin) / ($cmax - $cmin)*10)/10; }' you don't  have an iso-filled contour because the data is in vert shader.  For the MeshVisual in this case the data must be in the frag shader via v_vert_data = a_vert_data in the vert shader.

Reply:
But put nband and the banded flag in the colormap is a good idea.